mod/wet4/views/default/ajax/notif_dd.php

```diff
@@ -7,7 +7,14 @@
 
 @author: Nick
 */
+?>
+<script>
+$( "li.elgg-menu-item-notifications div.message-dd-holder div.message-dd-block a" ).first().focus(); //add focus on the first link of notifications
+
+$( "li.elgg-menu-item-messages div.message-dd-holder div.message-dd-block a" ).first().focus(); //add focus on the first link of messages
```

Comment:
This focus is on a element from the dropdown in a ajax call. Only one focus is use at the time.